### DISCUSSION TITLE
looking to learn how to pull requests for deauth 2.4g 5.g penetesting

### DISCUSSION THREAD

**Discussion by Glen7219** [community/community#64351] [New to GitHub]
### Select Topic Area

Show & Tell

### Body

Hi I'm public newbie looking to learn how to pull requests for deauth 2.4g 5.g penetesting and jamming for learning newbie here so me down softly thank tou

**Comment by Akash1134** [comment]
Welcome to GitHub and the GitHub Community @Glen7219  🎆 ! 

We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

I have moved your post to the relevant category so experts can look into it. 

Best of luck!
